.husky/_/husky.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env sh
```

Comment:
I think this file should be ignored? There's a `.gitignore` file in the `_` folder.

Reply:
I thought the same, but if its not commited, then everyone who clones the repo need to run `npx husky-init` to generate the script. If we commit it, then husky wil be running for everyone who clones the repo? Am I missing something?

Reply:
https://typicode.github.io/husky/#/?id=with-is-ci

that's what the `prepare` script it for

Reply:
I will keep the `prepare` script because I have also added it and remove the husky shell script then